### ISSUE TITLE
Delft integration - ModuleNotFoundError: No module named 'encodings'

### ISSUE THREAD

**Issue by lfoppiano** [kermitt2/grobid#536]
I was trying grobid with delft today but I got new issues: 

I'm using conda (I tried both with python 3.6 and 3.7): 

```
Could not find platform independent libraries <prefix>
Could not find platform dependent libraries <exec_prefix>
Consider setting $PYTHONHOME to <prefix>[:<exec_prefix>]
Fatal Python error: Py_Initialize: unable to load the file system codec
ModuleNotFoundError: No module named 'encodings'
```

It seems a problem with encodings, maybe is just related to my system... Cannot really say at this stage.

**Comment by lfoppiano** [comment]
OK, it was a specific problem to mac osx and not related to grobid itself.  
Anyway I solved by updating conda to version 4.8.1

**Comment by lfoppiano** [comment]
funny, funny, now I have the problem on linux and not on mac 😭
